### DISCUSSION TITLE
Can the Ghelper app set a lower CPU temperature limit to keep the fan from running?

### DISCUSSION THREAD

**Discussion by NatsuZoraKanata** [seerge/g-helper#2590] [Ideas]
Hey Everyone!
My laptop model is FA507XV. In silent mode (performance mode=2), the fan starts running when the CPU reaches 60°C. Currently, the maximum temperature limit in GHelper is 75°C. Can the maximum CPU temperature limit be lowered (e.g., to 59°C) to ensure that the cooling fan never runs?

**Comment by seerge** [comment]
@NatsuZoraKanata hello, you can extend a range by adding parameter to app config as explained here https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-windows-power-plans-with-each-mode

But keep in mind, there is no magic here. Temp limit is achieved by dropping your CPU power consumption.
